docs/source/proposals/jit-classes.rst

```diff
@@ -69,7 +69,7 @@ C structure::
         complex64 field2;
     };
 
-This will also be comptabile with an aligned NumPy structured dtype.
+This will also be compatible with an aligned numpy structure dtype.
```

Comment:
Seems like the rebase messed this one up. We recently merged https://github.com/numba/numba/pull/8144 and this change would revert some changes introduced by that one.

```suggestion
This will also be compatible with an aligned NumPy structure dtype.
```

Reply:
```suggestion
This will also be compatible with an aligned NumPy structured dtype.
```

Reply:
I think the second suggestion is the correct one, but see also: https://github.com/numba/numba/pull/8181#discussion_r904853467